app/src/test/java/com/kickstarter/viewmodels/DiscoveryFragmentViewModelTest.java

```diff
@@ -327,9 +378,10 @@ public void testStartEditorialActivity() {
 
     // Click on editorial
     this.vm.inputs.editorialViewHolderClicked(Editorial.GO_REWARDLESS);
+    this.vm.inputs.editorialViewHolderClicked(Editorial.LIGHTS_ON);
 
-    this.startEditorialActivity.assertValue(Editorial.GO_REWARDLESS);
-    this.koalaTest.assertValues("Discover List View", "Editorial Card Clicked");
+    this.startEditorialActivity.assertValues(Editorial.GO_REWARDLESS, Editorial.LIGHTS_ON);
+    this.koalaTest.assertValues("Discover List View", "Editorial Card Clicked", "Editorial Card Clicked");
```

Comment:
Question for @jamielynnroth, will there be Lake events for this project? Since the Koala events have been disabled.